bcda-site-static/decryption_walkthrough.md

```diff
@@ -40,21 +41,42 @@ You will be interacting with the [BCDA API](./user_guide.html) in your browser u
 - Open your browser and navigate to:
 
 ### [https://sandbox.bcda.cms.gov/api/v1/swagger](./api/v1/swagger/){:target="_blank"}
-<img src="assets/img/decrypt_demo_01.png" alt="Screenshot of swagger" width="400" />
+<img src="assets/img/decrypt_walkthrough_01.png" alt="Screenshot of swagger" width="400" />
 
-Provide your token to get full access to the API.
+## Getting a token
 
+Use your credentials to get an access token.
```

Comment:
Should this link to the public sandbox ones?  not sure it should, but worth considering.

Reply:
Maybe.  How about:
`Use your credentials to get an access token.  If you need credentials, use `[these](https://sandbox.bcda.cms.gov/user_guide.html#authentication-and-authorization).